expression/builtin_arithmetic_test.go

```diff
@@ -367,10 +367,10 @@ func (s *testEvaluatorSuite) TestArithmeticDivide(c *C) {
 		c.Assert(err, IsNil)
 		c.Assert(sig, NotNil)
 		switch sig.(type) {
-		case *builtinArithmeticIntDivideIntSig:
-			c.Assert(sig.PbCode(), Equals, tipb.ScalarFuncSig_IntDivideInt)
-		case *builtinArithmeticIntDivideDecimalSig:
-			c.Assert(sig.PbCode(), Equals, tipb.ScalarFuncSig_IntDivideDecimal)
+		case *builtinArithmeticDivideRealSig:
+			c.Assert(sig.PbCode(), Equals, tipb.ScalarFuncSig_DivideReal)
+		case *builtinArithmeticDivideDecimalSig:
+			c.Assert(sig.PbCode(), Equals, tipb.ScalarFuncSig_DivideDecimal)
```

Comment:
Suggest add "default" branch and raise failure, to avoid similar error in the future.